src/main.cpp

```diff
@@ -823,6 +844,7 @@ QString UsageArguments ( char **argv )
         "                        supported for headless server mode)\n"
         "  -n, --nogui           disable GUI\n"
         "  -p, --port            set your local port number\n"
+        "  -Q, --qos             set QoS: DF=0 AF31=104 CS4=128default EF=184\n"
```

Comment:
Can I ask for the help string over two lines with the first saying `request network quality of service` and the second giving the presets?

Reply:
- Help string is now two lines, where the first line is a description and the second line is a usage.
- This option is for those few that want to select a different QoS for their network.